vendor/github.com/moby/buildkit/session/content/attachable.go

```diff
@@ -0,0 +1,132 @@
+package content
+
+import (
+	"context"
+
+	api "github.com/containerd/containerd/api/services/content/v1"
+	"github.com/containerd/containerd/content"
+	"github.com/containerd/containerd/errdefs"
+	contentservice "github.com/containerd/containerd/services/content"
```

Comment:
@tonistiigi was this expected that this now pulls in containerd? (looks like this causes the containerd metadata dependencies to be pulled in, including bbolt)

Reply:
@AkihiroSuda I guess this is the new local cache importer. We should refactor it so that just importing client package doesn't import it.

@thaJeztah I think it is ok atm. We can create a tracking issue and fix it later.

Reply:
Opened https://github.com/moby/buildkit/issues/886 for tracking